@AH-64D Apache Longbow/Addons/fza_ah64_fire/CfgFunctions.hpp

```diff
@@ -0,0 +1,21 @@
+#ifdef __A3_DEBUG__
+#define R recompile = 1
+#else
+#define R recompile = 0
+#endif
+class CfgFunctions
+{
+    class fza_ah64_fire {
+        tag="fza_fire";
+        class functions {
+            file = "\fza_ah64_fire\functions";
+            class damageSystem {R;};
+            class damageEnginefire {R;};
+            class fireInit {R;};
```

Comment:
Why is this function called `fza_fire_fnc_fireInit` rather than just `fza_fire_fnc_init`